yarn-project/aztec.js/src/sandbox/index.ts

```diff
@@ -27,8 +25,8 @@ export const { PXE_URL = 'http://localhost:8080' } = process.env;
  */
 export function getSandboxAccountsWallets(pxe: PXE): Promise<AccountWalletWithPrivateKey[]> {
   return Promise.all(
-    zip(INITIAL_SANDBOX_ENCRYPTION_KEYS, INITIAL_SANDBOX_SIGNING_KEYS, INITIAL_SANDBOX_SALTS).map(
-      ([encryptionKey, signingKey, salt]) => getSchnorrAccount(pxe, encryptionKey!, signingKey!, salt).getWallet(),
+    INITIAL_SANDBOX_ENCRYPTION_KEYS.map((encryptionKey, i) =>
+      getSchnorrAccount(pxe, encryptionKey!, INITIAL_SANDBOX_SIGNING_KEYS[i]!, INITIAL_SANDBOX_SALTS[i]).getWallet(),
```

Comment:
Probably :)

I'd say if it's trivially implementable as above, don't use. I get `zip` is more descriptive (or is it? Someone might have to go read the lodash docs to understand what zip does) but it also just seems like bloat to add an entire dependency to achieve, even if it is a granular dependency. I think this is still perfectly readable and there's no hidden code being executed.

Lodash just seems like an old *javascript* focused frontend library (with debounce, and throttle etc and `get` style functions that take string based property paths). It's just not very... modern in it's approach. It was designed to fill a stdlib gap and provide browser compatibility. And deepClone stuff can be a footgun, only cloning plain data and not classes. I've seen when you put something like lodash into hands of developers it starts to get used *everywhere*, and looking through what lodash offers *most* of it is obsolete in a modern TS node.js environment.

I mean, take a look at this, and start clicking through the implementation:
https://github.com/lodash/lodash/blob/main/src/zip.ts

Do we really want that, to achieve the same thing as what I did above?

And what happened to version 5 that was meant to land 2 years ago?
https://github.com/lodash/lodash/wiki/Roadmap

I'd say if we have some clear code that can't be "done better" to avoid lodash, maybe a granular dependency can be warranted, but I'd want us to always try very hard to see if there is some neat technique in a given tempting case, to just avoid.

Reply:
(continuing just for the sake of the argument, I've already caved and submitted a PR that removes much lodash usage)

I agree that parts of lodash were meant to fill in gaps, like `flatMap`, `every`, or other functions that eventually made it. And I agree that `get`-style functions, while really popular in other languages, don't make much sense since optional chaining and nullish coalescing were introduced into the language, and probably don't play nicely with typescript (assuming no one abused string literal types to make it work). 

But I think there's value in many of the higher-order functions, as they improve readability (if you're used to it, granted), and most importantly they reduce bugs. Lodash is battle-tested enough that it's more likely that there's a bug in a seemingly simple function you wrote yourself, than if you pull a lodash dependency, not to mention the time saved.

Still, lodash if one of the few libraries that I trust like this. In pretty much any other scenario, I'd go with reimplementing, especially if it's production code.

Reply:
I'm not anti high level functions that there is a risk of bugging if we do it ourself.
Although, I'd probably check the implementation, and if it's mess of backwards compatible js, I would also be tempted to just lift the implementation out of lodash, strip it back to a bare TS implementation, and put it into an appropriately named foundation module.

You mentioned `times` in your PR. It's a one liner, and I agree it's a little verbose to type every time. This would be particularly easy to put into an `array` module in foundation for example.

Reply:
Done! https://github.com/AztecProtocol/aztec-packages/pull/3877